system-variables.md

```diff
@@ -364,13 +384,18 @@ Constraint checking is always performed in place for pessimistic transactions (d
 
 - Scope: SESSION | GLOBAL
 - Default value: 0
+- Range: [0, 2147483647]
 - When this value is greater than `0`, TiDB will batch commit statements such as `INSERT` or `LOAD DATA` into smaller transactions. This reduces memory usage and helps ensure that the `txn-total-size-limit` is not reached by bulk modifications.
 - Only the value `0` provides ACID compliance. Setting this to any other value will break the atomicity and isolation guarantees of TiDB.
 
 ### tidb_enable_1pc <span class="version-mark">New in v5.0</span>
 
+> **Note:**
+>
+> The default value of `ON` only applies to new clusters. if your cluster was upgraded from an earlier version of TiDB, the value `OFF` will be used instead.
```

Comment:
The default reading sequence is up -> down. So it's a bit weird that this note is presented before Line 398. What about writing it in Line 398, just after "Default value: ON"?

Reply:
I used this style to match the deprecated sysvars, where we have the warning before the variable options. But warning didn't seem appropriate, so I changed it to note.